src/Models/ExternalTool.cs

```diff
@@ -76,11 +125,70 @@ public void TryAdd(string name, string icon, string args, string env, Func<strin
 
             Founded.Add(new ExternalTool
             {
-                Name = name,
-                Icon = icon,
-                OpenCmdArgs = args,
-                Executable = path,
+                Name = name, Icon = icon, OpenCmdArgs = args, Executable = path, FallbackIcon = fallbackIcon
             });
         }
+
+        public void FindJetBrainsFromToolbox(Func<string> platform_finder)
```

Comment:
“platform_finder” I dont think that's a good naming style :(

Reply:
It's my mistake.